apps/ledger-live-mobile/src/families/ethereum/EditTransaction/Cancel.tsx

```diff
@@ -0,0 +1,64 @@
+import { fromTransactionRaw } from "@ledgerhq/live-common/families/ethereum/transaction";
```

Comment:
This file is just a middle man to craft the cancel transaction and then redirects directly to the SendSelectDevice screen